packages/upswyng-web/src/data-mocks.ts

```diff
@@ -0,0 +1,307 @@
+      city: "Longmont",
+      state: "CO",
+      zip: "80501",
+    },
+    createdAt: new Date("2019-12-10T08:47:04.164Z"),
+    deleted: false,
+    description:
+      "Life Choices helps women and men who are facing unplanned pregnancies, offers healing from after-abortion stress, and educates about sexual life choices.  At Life Choices our goal is to educate women on all their options so that each woman can make a choice that she is comfortable with. We come along side each woman to support her no matter what her choice may be. Whether a woman decides to carry her child to full term, go through the adoption process, or terminate the pregnancy, we will provide support.",
+    resourceId: "5def5b88376022bd304171ba",
+    kudos: 0,
+    lastModifiedAt: new Date("2019-12-10T08:47:04.164Z"),
+    latitude: 40.1815242,
+    legacyId: "-Kigar_wzpTiyh2mUEXy",
+    longitude: -105.09414649999997,
+    name: "Life Choices - Longmont",
+    phone: "303-651-2050",
+    schedule: { alwaysOpen: false, timezone: "America/Denver", _items: [] },
+    services: ["Free Pregnancy", "Ultrasound", "STI Testing"],
+    subcategories: [],
+    website: "http://www.lifechoices.org",
+  },
+  {
+    _id: "5def5b885fe02cacaec191de",
+    address: {
+      address1: "PO Box 270181",
+      city: "Louisville",
+      state: "CO",
+      zip: "80027",
+    },
+    createdAt: new Date("2019-12-10T08:47:04.293Z"),
+    deleted: false,
+    description:
+      "The Little Flower Maternity Home is a home where pregnant mothers over 18, can come and live free of charge from any point in their pregnancy until their baby is 6 months old.  The Little Flower is not a hand-out, but rather a hand-up to these homeless mothers in need.    Mothers at The Little Flower spend their time saving money and working towards self-sufficiency.  Staff members and Volunteers guide and encourage her on her way toward self-sufficiency.   This gives her the opportunity to focus on having a safe & healthy pregnancy and to set future goals for her baby and herself.  The Little Flower provides a safe, healthy and loving community environment.  Mothers can enter the next chapter of their lives, leaving behind unhealthy and unsafe behavior. Basic Qualifications for Mothers: Pregnant, At least 18 years old, Currently Drug and Alcohol free, Free from immediate physical danger, Free from diagnosed severe mental illness, Without other children in her care",
+    resourceId: "5def5b88376022bd304171c6",
+    kudos: 0,
+    lastModifiedAt: new Date("2019-12-10T08:47:04.293Z"),
+    latitude: 39.977763,
+    legacyId: "-KmOFmcStxVNB-5Tszdx",
+    longitude: -105.13192960000003,
+    name: "The Little Flower Maternity Home",
+    phone: "(720) 609-2934",
+    schedule: { alwaysOpen: false, timezone: "America/Denver", _items: [] },
+    services: ["Maternity housing"],
+    subcategories: [],
+    website: "http://littleflowermaternity.org",
+  },
+] as TResource[];
```

Comment:
So this is a bit backwards. What you want is to have `const mockResources: TResource[] = ....`. With that, if some field is missing then the fake data will fail lint. As it is, this says "no matter what's in this array... they're `TResource`s ... even if they aren't`

Reply:
Same with the category stuff below.

Reply:
ah yup, that's a total derp. [updated](https://github.com/CodeForBoulder/upswyng/pull/265/commits/b7d84185aa062f2e5a7d120335d1c723a5968330)